lib/internal/bootstrap/loaders.js

```diff
@@ -160,9 +160,13 @@ const schemelessBlockList = new SafeSet([
   };
 }
 
+/**
+ * @typedef {import('../../../typings/bindings').InternalBindingMap} InternalBindingMap
+ */
 // Set up internalBinding() in the closure.
+// Do not use `internalBinding` variable name due to the breakage of TypeScript autocompletion.
```

Comment:
I don't understand this comment.

Reply:
That's definitely an outdated comment. I'll remove it!